projects/1.20-fabric/assets/more-wizards/morewizardsmod/lang/zh_cn.json

```
@@ -0,0 +1,41 @@
+{
+  "item.morewizardsmod.more_wizards_lightning_spell_book": "雷电魔法书",
+  "item.morewizardsmod.more_wizards_soul_spell_book": "灵魂魔法书",
+  "item.morewizardsmod.lightning_staff": "雷电权杖",
+  "item.morewizardsmod.netherite_lightning_staff": "下界合金雷电权杖",
+  "item.morewizardsmod.soul_staff": "灵魂权杖",
+  "item.morewizardsmod.netherite_soul_staff": "下界合金灵魂权杖",
+
+  "item.morewizardsmod.lightning_wizard_armor_t1_head": "雷电学徒帽",
+  "item.morewizardsmod.lightning_wizard_armor_t1_chest": "雷电学徒长袍 ",
+  "item.morewizardsmod.lightning_wizard_armor_t1_legs": "雷电学徒护腿",
+  "item.morewizardsmod.lightning_wizard_armor_t1_feet": "雷电学徒靴子",
+  "item.morewizardsmod.lightning_wizard_armor_t2_head": "雷电魔法师帽",
+  "item.morewizardsmod.lightning_wizard_armor_t2_chest": "雷电魔法师长袍",
+  "item.morewizardsmod.lightning_wizard_armor_t2_legs": "雷电魔法师护腿",
+  "item.morewizardsmod.lightning_wizard_armor_t2_feet": "雷电魔法师靴子",
+
+  "item.morewizardsmod.soul_wizard_armor_t1_head": "灵魂学徒帽",
+  "item.morewizardsmod.soul_wizard_armor_t1_chest": "灵魂学徒长袍",
+  "item.morewizardsmod.soul_wizard_armor_t1_legs": "灵魂学徒护腿",
+  "item.morewizardsmod.soul_wizard_armor_t1_feet": "灵魂学徒靴子",
+  "item.morewizardsmod.soul_wizard_armor_t2_head": "灵魂魔法师帽",
+  "item.morewizardsmod.soul_wizard_armor_t2_chest": "灵魂魔法师长袍",
+  "item.morewizardsmod.soul_wizard_armor_t2_legs": "灵魂魔法师护腿",
+  "item.morewizardsmod.soul_wizard_armor_t2_feet": "灵魂魔法师靴子",
+
+  "spell.morewizardsmod.lightning_strike.name": "雷霆一击",
+  "spell.morewizardsmod.lightning_strike.description": "召唤闪电束劈击一个目标3次",

```

```suggestion
  "spell.morewizardsmod.lightning_strike.description": "召唤闪电束劈击目标3次",
```

既然可以自动追踪（不定量的目标），那就不需要冠词了
`召唤`是润出来的，可以换成`使用`
目标还是敌人你自己选一个，保持上下文统一且读着舒服就行